iFixit/react-commerce
# feat: Add price filter

This PR adds a price range filter to the collection page.

A few other improvements are also introduced by this PR:

- Simplified filter ids: now root filter ids are the facet names, so its easier to clear filters by facet name
- Added clear buttons below filter sections
- Refactored lib folder

## QA

1. Visit the Vercel PR preview
2. Visit any collection page (other than `/parts` and `/tools`)
3. Verify that when a price range is selected products are filtered accordingly
4. Verify that a custom range can be inserted and products are filtered accordingly

Closes #18
